pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -545,6 +548,10 @@ type HTTPConfig struct {
 	// Optional proxy URL.
 	// +optional
 	ProxyURL string `json:"proxyURL,omitempty"`
+	// FollowRedirects specifies whether the client should follow HTTP 3xx redirects.
+	// +optional
+	// +kubebuilder:default:=true
+	FollowRedirects bool `json:"followRedirects"`
```

Comment:
https://github.com/prometheus-operator/prometheus-operator/pull/4563 uses a *bool field instead. This has the advantage to delegate the handling of the default value to Prometheus.